setup.py

```diff
@@ -2,19 +2,28 @@
 #
 # Python setup for Qiling framework
 
-
+import sys, os
 from setuptools import setup, find_packages
 
-from qiling import __version__ as ql_version
+here = os.path.abspath(os.path.dirname(__file__))
+gb = {}
+with open(os.path.join(here, "qiling", "__version__.py"), "r+") as f:
+    exec(f.read(), gb)
 
-VERSION = ql_version
+VERSION = gb['__version__']
 
 with open('requirements.txt') as f:
     required = f.read().splitlines()
 
 with open("README.md", "r", encoding="utf-8") as ld:
     long_description = ld.read()
 
+if "linux"  in sys.platform:
```

Comment:
can we move all the requirement here ? and not setup.py and requirement.txt

Reply:
See commits 4818261, 3c6b61b and 81fca85.